core/platform/models.py

```diff
@@ -113,59 +151,126 @@ def import_email_services(cls):
 
     @classmethod
     def import_memcache_services(cls):
+        """Imports and returna gae_memcache_services.
+
+        Returns:
+            module. The gae_memcache_services module.
+        """
         from core.platform.memcache import gae_memcache_services
         return gae_memcache_services
 
     @classmethod
     def import_taskqueue_services(cls):
+        """Imports and returns gae_taskqueue_services.
+
+        Returns:
+            module. The gae_taskqueue_services module.
+        """
         from core.platform.taskqueue import gae_taskqueue_services
         return gae_taskqueue_services
 
     @classmethod
     def import_search_services(cls):
+        """Imports and returns gae_search_services.
+
+        Returns:
+            module. The gae_search_services module.
+        """
         from core.platform.search import gae_search_services
         return gae_search_services
 
     NAME = 'gae'
 
 
 class Registry(object):
+    """Platform-agnostic interface for retrieving platform-specific modules.
+    """
+
+    # Maps platform names to the corresponding module registry classes.
     _PLATFORM_MAPPING = {
         _Gae.NAME: _Gae,
     }
 
     @classmethod
     def _get(cls):
+        """Method to retrieve the _Gae Model.
```

Comment:
_Gae is not a model. It is a class.

`Returns the appropriate interface class for platform-specific imports.`